### PR TITLE
Zoom source doc update

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/webhook-sources/zoom.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ Follow the below steps to get the required fields for user configuration:
     - Developer Email Address 
   <br/><img src={useBaseUrl('img/send-data/zoom-app-info.png')} alt="zoom-app-info" width="500"/>
 1. Click **Continue**, to copy the **Secret Token** value. <br/><img src={useBaseUrl('img/send-data/secret-token.png')} alt="secret-token" width="500"/>
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/webhook-sources/zoom.md:42]:
```suggestion
1. Click **Continue** to copy the **Secret Token** value. <br/><img src={useBaseUrl('img/send-data/secret-token.png')} alt="secret-token" width="500"/>
```

**Review by JainM6 — DISMISSED** [review]
Minor fix

File: docs/send-data/hosted-collectors/webhook-sources/zoom.md [hunk 2]
```diff
@@ -69,7 +70,7 @@ To configure a Zoom Source:
 
 ## Enable Events Subcriptions
 
-1. Go back to the **Features** tab in the Zoom Marketplace.
+1. Go back to the page in the Zoom Marketplace which you left open in the step 1(#setup-and-configuration).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/webhook-sources/zoom.md:73]:
```suggestion
1. Go back to the page in the Zoom Marketplace that you left open in [step 1](#setup-and-configuration).
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
quick fix

**Review by kimsauce — APPROVED** [review]

**Pull request merged**